workspace_tools/export/iar_mts_dragonfly_f411re.ewp.tmpl

```diff
@@ -593,11 +593,11 @@
         </option>
         <option>
           <name>OCOutputOverride</name>
-          <state>0</state>
+          <state>1</state>
         </option>
         <option>
           <name>OOCOutputFile</name>
-          <state>{{name}}.bin</state>
+          <state>application.bin</state>
```

Comment:
The predefined application name is referenced by the post-build script, which combines the bootloader and application binaries into a single image.  I was unable to figure out how to reference {{name}} (or find out if it is stored as an IAR environment variable) from my post-build batch script.